st2common/bin/migrations/v2.1/st2-migrate-runners.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+from eventlet.green import subprocess
+
+from st2common.util.green.shell import run_command
+
+
+def main():
+    timeout = 180
+
+    args = [
+        'st2ctl',
+        'reload',
+        '--register-all',
```

Comment:
This would probably be better and fast, since if user has a lot of resources it can take a long time.